Specs/Scene/WebMapTileServiceImageryProviderSpec.js

```diff
@@ -86,6 +86,57 @@ describe("Scene/WebMapTileServiceImageryProvider", function () {
     expect(parseInt(queryObject.tilerow, 10)).toEqual(tilerow);
   });
 
+  it("generates expected tile urls for subdomains", function () {
+    var options = {
```

Comment:
This tests doesn't validate that the `{s}` is actually replaced with the default subdomains `[a, b,c]`, it validates that you are in KVP mode, but I think we should add an `expect` to check the URL string to make sure the template arguments are set correctly.